examples/model_selection/plot_randomized_search.py

```diff
@@ -23,19 +23,20 @@
 import numpy as np
 
 from time import time
-from scipy.stats import randint as sp_randint
+import scipy.stats as stats
+from sklearn.utils.random import loguniform
 
-from sklearn.model_selection import GridSearchCV
-from sklearn.model_selection import RandomizedSearchCV
+from sklearn.model_selection import GridSearchCV, RandomizedSearchCV
 from sklearn.datasets import load_digits
-from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import SGDClassifier
```

Comment:
I assume SVC didn't work well or was too slow?

Reply:
@stsievert, a comment here?

Reply:
I think the contrast with l1_ratio is relatively compelling.